src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -1,5 +1,10 @@
 // Packages
 import React, { Component } from 'react';
+import Ratings from 'react-ratings-declarative';
+import Cookies from 'universal-cookie';
+import PropTypes from 'prop-types';
+
+/* recharts */
```

Comment:
Rename it to something generic, so that we can use it for all other dependencies.  `react-ratings-declarative, universal-cookie` can also be added in this section

Reply:
Yes